pkg/app/piped/platformprovider/kubernetes/applier.go

```diff
@@ -81,6 +92,17 @@ func (a *applier) CreateManifest(ctx context.Context, manifest Manifest) error {
 		return a.initErr
 	}
 
+	if a.input.AutoCreateNamespace == true {
+		err := a.kubectl.CreateNamespace(
+			ctx,
+			a.platformProvider.KubeConfigPath,
+			a.getNamespaceToRun(manifest.Key),
+		)
+		if err != nil && !errors.Is(err, errReplaceAlreadyExists) {
```

Comment:
Rename due to https://github.com/pipe-cd/pipecd/pull/4504#discussion_r1271260320